src/platform/mt8195/lib/dai.c

```diff
@@ -0,0 +1,20 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2021 Mediatek
+ *
+ * Author: YC Hung <yc.hung@mediatek.com>
+ */
+
+#include <sof/common.h>
+#include <sof/lib/dai.h>
+#include <sof/lib/memory.h>
+#include <sof/sof.h>
+#include <sof/spinlock.h>
+#include <ipc/dai.h>
+#include <ipc/stream.h>
+
+
+int dai_init(struct sof *sof)
+{
+	return 0;
+}
```

Comment:
The fact that this file is empty and has redundant includes feels odd to me. Are you sure this is correct?